test/polygons/conversion.py

```diff
@@ -0,0 +1,66 @@
+# Usage:
+# python conversion.py file.cpp output.txt
```

Comment:
Do we need this script anymore if all the old test files have already been converted?

Reply:
We don't need this if we dump the new format directly.